datadoc_json_schema/datadoc_json_schema.json

```
@@ -0,0 +1,376 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "definitions": {
+        "languageStringType": {
+            "$comment": "Reusable langugage string type",
+            "type": "object",
+            "properties": {
+                "en": {
+                    "type": "string"
+                },
+                "nn": {
+                    "type": "string"
+                },
+                "nb": {
+                    "type": "string"
+                }
+            },
+            "required": [
+                "nb"
+            ]
+        },
+        "temporalityTypeType": {
+            "$comment": "Reusable temporalitytype type",
+            "type": "string",
+            "enum": [
+                "FIXED",
+                "STATUS",
+                "ACCUMULATED",
+                "EVENT"
+            ]
+        }
+    },
+    "type": "object",
+    "properties": {
+        "percentage_complete": {
+            "$comment": "Used by the DataDoc app (internal information)",
+            "type": "integer"
+        },
+        "document_version": {
+            "$comment": "DataDoc model version of this Json Schema",
+            "const": "1.0.0"
```

```suggestion
            "const": "2.0.0"
```

This shouldn't shadow the existing model version IMO, there are breaking changes here in addition to changing the single source of truth.